tests/util/test_yaml.py

```diff
@@ -162,13 +172,13 @@ def get_password(self, domain, name):
 
 class TestSecrets(unittest.TestCase):
     """Test the secrets parameter in the yaml utility."""
+    # pylint: disable=protected-access,invalid-name
```

Comment:
Ah alrighty!

On Sat, Sep 3, 2016 at 1:44 PM, Johann Kellerman <notifications@github.com>
wrote:

> In tests/util/test_yaml.py
> <https://github.com/home-assistant/home-assistant/pull/3159#discussion_r77442018>
> :
>
> > @@ -162,13 +172,13 @@ def get_password(self, domain, name):
> >
> >  class TestSecrets(unittest.TestCase):
> >      """Test the secrets parameter in the yaml utility."""
> > +    # pylint: disable=protected-access,invalid-name
>
> Tests are excluded from linting, but I prefer to fix them as I work on them
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/3159/files/d314ae6a6c3a7dda65a97b7154a812f0c91c0882#r77442018>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZYA-UWXSa-Nr6Wqm911eviJzWOF9ks5qmc4hgaJpZM4J0Uoq>
> .
>